app/models/ext_management_system.rb

```diff
@@ -86,6 +86,7 @@ def self.create_from_params(params, endpoints, authentications)
   has_many :networks,          :through => :hardwares
   has_many :disks,             :through => :hardwares
   has_many :physical_servers,  :foreign_key => :ems_id, :inverse_of => :ext_management_system, :dependent => :destroy
+  has_many :physical_server_profiles, :foreign_key => :ems_id, :inverse_of => :ext_management_system, :dependent => :destroy
```

Comment:
Just a style thing, but easier on the eye to align these and maybe split it off from the group above:
```suggestion

  has_many :physical_servers,         :foreign_key => :ems_id, :inverse_of => :ext_management_system, :dependent => :destroy
  has_many :physical_server_profiles, :foreign_key => :ems_id, :inverse_of => :ext_management_system, :dependent => :destroy
```

Reply:
Fixed in the latest commit.